package/doc/sphinx/source/conf.py

```diff
@@ -156,8 +158,15 @@
     'titles_only': False,
 }
 
+html_context = {
+    'versions_json_url': 'https://mdanalysis.github.io/mdanalysis/versions.json'
```

Comment:
Does this have to be the true URL or could it be our CNAME, for consistency?

Reply:
It can be any valid address, so I've changed it to docs.mdanalysis.org :-)